sdcm/cluster.py

```diff
@@ -2883,6 +2938,21 @@ def __init__(self, ec2_ami_id, ec2_subnet_id, ec2_security_group_ids,
         self.nemesis_threads = []
         self.termination_event = threading.Event()
 
+    def populate_db(self, node):
+        node.remoter.run("sudo rm -rf /var/lib/cassandra/*")
+        node.remoter.run("sudo cp -a /root/* /var/lib/cassandra/")
+        node.remoter.run("sudo rm -rf /var/lib/cassandra/commitlog/*")
+        node.remoter.run("sudo mkdir /raid0/cassandra/logs")
+        node.remoter.run("sudo chmod -R 777 /raid0/cassandra")
+        node.remoter.run("sudo chown -R cassandra.cassandra /raid0/cassandra")
```

Comment:
`colon` between username and group name is better?
'chown username.groupname ...' doesn't work in MAC OSx.